pkg/eval/builtin_fn_str_test.go

```diff
@@ -39,6 +39,9 @@ func TestBase(t *testing.T) {
 		That(`base 16 42 233`).Puts("2a", "e9"),
 		That(`base 1 1`).Throws(ErrBadBase),
 		That(`base 37 10`).Throws(ErrBadBase),
+		That(`base 16 18446744073709551607`).Puts("fffffffffffffff7"),
+		That(`base 10 "0xfffffffffffffff7"`).Puts("18446744073709551607"),
+		That(`base 8 (num 1/8)`).Throws(ErrInvalidInput),
```

Comment:
You should also add the test I proposed (or something very similar): `That(`base (num 16) (math:pow 2 8)`).Puts( "100")`.

Reply:
Done.